test/e2e/node/pod_resize.go

```diff
@@ -1440,8 +1442,135 @@ func doPodResizeErrorTests() {
 	}
 }
 
+func doPodResizeSchedulerTests() {
+	f := framework.NewDefaultFramework("pod-resize-scheduler")
+
+	ginkgo.It("pod-resize-scheduler-tests", func() {
+		nodes, _ := f.ClientSet.CoreV1().Nodes().List(context.TODO(), metav1.ListOptions{})
+		nodeNum := len(nodes.Items)
+		framework.ExpectEqual(nodeNum, 1)
+
+		ginkgo.By("Checking the node allocatable CPU resources!")
+		node := nodes.Items[0]
+		nodeAllocatableResources := node.Status.Allocatable
```

Comment:
Please see my comments in PR https://github.com/vinaykul/kubernetes/pull/11

Assuming the PR is a go, I'm fine with merging this tomorrow if we don't get exception and then fixing the test before test freeze.